<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>507</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>627</width>
    <height>507</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>627</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MagChain Input Script Editor ver 0.1</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonEquil">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>equilibration</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTimestep">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRestart">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>restart every</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRestart">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRun">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditYBox">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHistogram">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistogram">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>histogram every</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrajectory">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>trajectory every</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOk">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStats">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditXBox">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelStats">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>statistics every</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImportInput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Input File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHistogramSize">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRestartFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>restart simulation from restart file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimestep">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>timestep *</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonProd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>production</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRun">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>simulation run *</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditZBox">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTrajectory">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRestartFileWarning">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAttradiusFileWarning">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>empty</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonFromRestart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>attraction radius from restart file</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupAttradius</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonFromExternal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>attraction radius from external file</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupAttradius</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="checkBoxAttradiusFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>load external attradius file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearAll">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelXBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>X box *</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelYBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Y box *</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Z box *</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistogramSize">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>histogram bin size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>20</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParticles">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParticles">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>10</y>
      <width>134</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>number of particles *</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowValues">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>119</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>show values</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAttradiusRestartSize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAttradiusExternalSize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonExportInfile">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>export input script</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupExport</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonAll">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>export All Project</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupExport</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditParticles</tabstop>
  <tabstop>lineEditXBox</tabstop>
  <tabstop>lineEditYBox</tabstop>
  <tabstop>lineEditZBox</tabstop>
  <tabstop>lineEditTimestep</tabstop>
  <tabstop>lineEditRun</tabstop>
  <tabstop>lineEditStats</tabstop>
  <tabstop>lineEditTrajectory</tabstop>
  <tabstop>lineEditRestart</tabstop>
  <tabstop>lineEditHistogram</tabstop>
  <tabstop>lineEditHistogramSize</tabstop>
  <tabstop>radioButtonEquil</tabstop>
  <tabstop>radioButtonProd</tabstop>
  <tabstop>pushButtonClearAll</tabstop>
  <tabstop>pushButtonImportInput</tabstop>
  <tabstop>checkBoxRestartFile</tabstop>
  <tabstop>checkBoxAttradiusFile</tabstop>
  <tabstop>radioButtonFromRestart</tabstop>
  <tabstop>radioButtonFromExternal</tabstop>
  <tabstop>lineEditAttradiusRestartSize</tabstop>
  <tabstop>pushButtonShowValues</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupAttradius"/>
  <buttongroup name="buttonGroupExport"/>
 </buttongroups>
</ui>
